scico/test/linop/test_circconv.py

```diff
@@ -160,6 +158,21 @@ def test_center(self, center):
             shift = -center[0]
         np.testing.assert_allclose(A @ x, snp.roll(B @ x, shift), atol=1e-5)
 
+    def test_fractional_center(self):
+        """A fractional center should keep outputs real"""
+        x, _ = uniform(minval=-1, maxval=1, shape=(4, 5), key=self.key)
+        h, _ = uniform(minval=-1, maxval=1, shape=(2, 2), key=self.key)
```

Comment:
It would be a bit better to not discard the key returned by the first call to `unform`, instead using it as the second key.